packages/web-config-server/src/apiV1/organisationUnitSearch.js

```diff
@@ -42,7 +42,12 @@ export default class extends RouteHandler {
     const project = await this.fetchProject();
     const projectEntity = await project.entity();
 
-    const allEntities = await projectEntity.getDescendants(project.entity_hierarchy_id);
+    const allEntities = await projectEntity.getDescendants(project.entity_hierarchy_id, {
+      type: {
+        comparator: 'not in',
+        comparisonValue: this.models.entity.typesExcludedFromWebFrontend,
+      },
+    });
```

Comment:
This query is the same with the other one in `organisationUnit.js`, does it make sense to extract them in a common method (eg `getDescendantsForFrontEnd`)?

From one hand, we shouldn't go very far re-defining methods for every param combination, on the other this encapsulates specific business logic so it's probably worth it. I will leave the decision to you! 😎

Reply:
Yeah it's a question I thought about, but only for 30 seconds, and then decided it's simple enough to repeat both places (and actually reads fairly fluently to my eyes)